File: bratek20-spring/src/test/java/pl/bratek20/spring/app/SpringAppTest.java
package pl.bratek20.spring.app;

import org.junit.jupiter.api.Test;
import org.springframework.context.annotation.Configuration;

import static org.assertj.core.api.Assertions.assertThatCode;

class SpringAppTest {

    @Configuration
    static class EmptyConfig {
    }

    @Test
    void shouldStartAppWithoutWebServlet() {
        SpringApp.run(EmptyConfig.class, new String[]{});

        assertThatCode(() -> SpringApp.run(EmptyConfig.class, new String[]{}))
            .doesNotThrowAnyException();
    }
}

File: bratek20-spring/src/test/java/pl/bratek20/spring/app/SpringAppTest.kt
package pl.bratek20.spring.app

import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Test
import org.springframework.context.annotation.Configuration

class SpringAppTest {

    @Test
    fun shouldStartAppWithoutWebServlet() {
        SpringApp.run()

        Assertions.assertThatCode { SpringApp.run() }
            .doesNotThrowAnyException()
    }
}